modules/Bio/EnsEMBL/Analysis/Hive/Config/Projection_subpipeline_conf.pm

```diff
@@ -0,0 +1,518 @@
+    'user'                      => '', # write db user
+    'password'                  => '', # password for write db user
+    'pipe_db_server'            => '', # host for pipe db
+    'dna_db_server'             => '', # host for dna db
+    'databases_server'          => '', # host for general output dbs
+    'pipe_db_port'              => '', # port for pipeline host
+    'dna_db_port'               => '', # port for dna db host
+    'databases_port'            => '', # port for general output db host
+    'uniprot_set'               => '', # e.g. mammals_basic, check UniProtCladeDownloadStatic.pm module in hive config dir for suitable set,
+    'output_path'               => '', # Lustre output dir. This will be the primary dir to house the assembly info and various things from analyses
+    'use_genome_flatfile'       => '1',# This will read sequence where possible from a dumped flatfile instead of the core db
+    'skip_projection'           => '0', # Will skip projection process if 1
+
+########################
+# Pipe and ref db info
+########################
+
+    'projection_source_db_name'         => '', # This is generally a pre-existing db, like the current human/mouse core for example
+    'projection_source_db_server'       => 'mysql-ens-mirror-1',
+    'projection_source_db_port'         => '4240',
+    'projection_source_production_name' => '',
+
+    'compara_db_name'   => 'leanne_ensembl_compara_95',
+    'compara_db_server' => 'mysql-ens-genebuild-prod-5',
+    'compara_db_port'   => 4531,
+
+    # The following might not be known in advance, since the come from other pipelines
+    # These values can be replaced in the analysis_base table if they're not known yet
+    # If they are not needed (i.e. no projection or rnaseq) then leave them as is
+    'pipe_db_name'  => $self->o('dbowner').'_'.$self->o('production_name').'_pipe_'.$self->o('release_number'),
+
+    'projection_lastz_db_name'     => $self->o('pipe_db_name'),
+    'projection_lastz_db_server'   => $self->o('pipe_db_server'),
+    'projection_lastz_db_port'     => $self->o('pipe_db_port'),
+
+    'dna_db_name'   => $self->o('dbowner').'_'.$self->o('production_name').'_core_'.$self->o('release_number'),
+
+    'projection_db_server'  => $self->o('databases_server'),
+    'projection_db_port'    => $self->o('databases_port'),
+
+    databases_to_delete => ['projection_db',],
+
+######################################################
+#
+# Mostly constant settings
+#
+######################################################
+
+    genome_dumps  => catdir($self->o('output_path'), 'genome_dumps'),
+    # This one is used by most analyses that run against a genome flatfile like exonerate, genblast etc. Has slice name style headers. Is softmasked
+
+    'primary_assembly_dir_name'              => 'Primary_Assembly',
+    'refseq_cdna_calculate_coverage_and_pid' => '0',
+    'contigs_source'                         => 'ena',
+
+    'min_toplevel_slice_length'   => 250,
+
```

Comment:
This does not seem to be needed, it can be removed

Reply:
Agreed, gone.